source/dynk.f90

```diff
@@ -2503,7 +2556,36 @@ real(kind=fPrec) function dynk_getvalue(element_name, att_name)
           goto 100 ! ERROR
         end if
 
-        ! Not yet supported : Multipoles (11)
+
+      case(11)
+      im=irm(ii)
+      if(att_name=="scaleall") then
+
+      else if(att_name(1:1) == "a" .or. att_name(1:1) == "b") then
+        if(LEN_TRIM(att_name) .eq. 5) then
+           range = 3
+          read(att_name(2:2), *) orderMult
```

Comment:
Don't use read() for casting numerical values. These should be read with the chr_cast interface in module string_tools. In general: for float values ensures proper rounding, and for integers and floats it allows for error checking and capturing.